CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 * Add Termux support [\#4749](https://github.com/rvm/rvm/pull/4749)
 
 #### New interpreters
-* Add support for TruffleRuby 19.3.0.
+* Add support for TruffleRuby 19.3.0
```

Comment:
Where are the PR references?

Reply:
Those entries were created by `update-truffleruby.rb`, so they don't have it (https://github.com/rvm/rvm/pull/4576)

Reply:
Can we add them retrospectively and remember them next time?

Reply:
@eregon?

Reply:
Sure, I can do it if there is a PR.
For 19.3.0 there was no PR though, I wanted to make the release public in a short time after the files went online, for all Ruby installers.